src/components/Download/index.jsx

```diff
@@ -0,0 +1,90 @@
+import ThemedImage from "@theme/ThemedImage";
+import React from "react";
+import styles from "./styles.module.css";
+
+export default function Download() {
+  const downloads = [
+      {
+          arch: "arm64 (Apple Silicon)",
+          dark: "img/download/apple-dark.png",
+          href: "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh",
+          light: "img/download/apple-light.png",
+          os: "macOS",
+      },
+      {
+          arch: "x86_64 (Intel)",
+          dark: "img/download/apple-dark.png",
+          href: "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh",
+          light: "img/download/apple-light.png",
+          os: "macOS",
+      },
+      {
+          arch: "x86_64 (amd64)",
+          dark: "img/download/linux-dark.png",
+          href: "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh",
+          light: "img/download/linux-light.png",
+          os: "Linux",
+      },
+      {
+          arch: "aarch64 (arm64)",
+          dark: "img/download/linux-dark.png",
+          href: "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh",
+          light: "img/download/linux-light.png",
+          os: "Linux",
+      },
+      {
+          arch: "ppc64le (POWER8/9)",
+          dark: "img/download/linux-dark.png",
+          href: "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh",
+          light: "img/download/linux-light.png",
+          os: "Linux",
+      },
+      {
+          arch: "x86_64",
+          dark: "img/download/windows-dark.png",
+          href: "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe",
+          light: "img/download/windows-light.png",
+          os: "Windows",
+      }
+  ];
+  return (
+    <div className={[styles.header, styles.section_padding].join(" ")}>
+        <div className={styles.header_image}>
+            <ThemedImage
+                alt="3D-Anvil illustration for conda-forge"
+                sources={{
+                    light: useBaseUrl("/img/anvil-light.svg"),
+                    dark: useBaseUrl("/img/anvil-dark.svg"),
+                }}
+            />
+        </div>
+        <div className={styles.header_content}>
+          {downloads.map(({ arch, dark, href, light, os }, index) => (
+            <Link to={href} key={index}>
+                <div className={styles.cardWrapper}>
+                  <div className={styles.card}>
+                    <ThemedImage
+                      className={styles.image}
+                      alt={`${os} logo`}
+                      title={`Download miniforge installer for ${os} ${arch}`}
+                      sources={{
+                        dark: useBaseUrl(`${dark}`),
+                        light: useBaseUrl(`${light}`),
+                      }}
```

Comment:
Since they are monochrome, would it be better to use `filter: invert(1)` CSS here and save one image?

Reply:
Clever!